qubes/mgmt.py

```diff
@@ -534,3 +570,95 @@ def vm_kill(self):
         assert not self.arg
         self.fire_event_for_permission()
         yield from self.dest.kill()
+
+    @api('mgmt.Events', no_payload=True)
```

Comment:
Yes, it is expected. Not sure explicitly documented, but this is exactly the same as for all other methods - it does anything only after sending EOF.